modules/affile/affile-dest.c

```diff
@@ -168,6 +169,9 @@ affile_dw_reopen(AFFileDestWriter *self)
     }
   else
     {
+      if (open_result == FILEOPENER_RESULT_ERROR_PERMANENT)
```

Comment:
this should be an "else if (open_result == FILEOPENER_RESULT_ERROR_PERMANENT) {} else {}" block, e.g.
the else and if should be combined, makes the flow more readable.

Reply:
Thank You!